fed/mosaic-sumo/src/test/java/org/eclipse/mosaic/fed/sumo/traci/commands/LaneGetLengthTest.java

```diff
@@ -28,20 +28,20 @@
 
     @Test
     public void execute() throws Exception {
-        String edgeId = "2_6_5_6";
+        String edgeId = "2_6_5";
         int laneIndex = 0;
 
 
         // RUN
         double length = new LaneGetLength().execute(traci.getTraciConnection(), edgeId, laneIndex);
 
         // ASSERT
-        assertEquals(1106.3, length, 0.1);
+        assertEquals(1106.4, length, 0.1);
```

Comment:
Did the lane length change? Or was the test a little bit inaccurate beforehand?

Reply:
Re-exporting the network resulted in slightly different shapes and lengths of the lanes.